client/src/components/Draftail/Draftail.scss

```diff
@@ -75,6 +78,20 @@ $draftail-editor-font-family: $font-sans;
 
 .Draftail-Editor {
   --draftail-offset-inline-start: 0;
+  position: relative;
+
+  // Adds additional hoverable area so block trigger will be shown when hovering to the side of the field.
+  &::after {
```

Comment:
Could also try styling the block toolbar div? Or padding + negative margin?

Reply:
Great suggestion! I've settled on styling the block toolbar div instead as you suggested